HTAN/Python Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb

```diff
@@ -49,7 +49,7 @@
     {
       "cell_type": "markdown",
       "source": [
-        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/isb-cgc/Community-Notebooks/blob/master/HTAN/Python%20Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb)"
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ncihtan/Community-Notebooks/blob/update-single-cell-python-notebook/HTAN/Python%20Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb)"
```

Comment:
Just checking here, should this be isb-cgc?

Reply:
Good catch! changed to master branch of isb-cgc
(cc @PozhidayevaDarya )

Reply:
Thank you!!